services/state/recordcache/recordcache.proto

```diff
@@ -53,3 +54,34 @@ message TransactionRecordEntry {
      */
     TransactionRecord transaction_record = 3;
 }
+/**
+ * As a single transaction is handled a receipt is created. It is stored in state for a configured time
+ * limit (perhaps, for example, 3 minutes). During this time window, any client can query the node and get the
+ * receipt for the transaction. The TransactionReceiptEntry is the object stored in state with this information.
+ */
+message TransactionReceiptEntry {
+    /**
+    * The ID of the node that submitted the transaction to consensus. The ID is the ID of the node as known by the
+    * address book. Valid node IDs are in the range 0..2^63-1, inclusive.
+    */
+    int64 node_id = 1;
```

Comment:
This should be `uint64` to match what [the `Node` object in state](https://github.com/hashgraph/hedera-protobufs/blob/main/services/state/addressbook/node.proto#L46) uses.

Reply:
Fixed. Thanks @jsync-swirlds